envoy/network/socket.h

```diff
@@ -73,6 +73,11 @@ class SocketAddressProvider {
    */
   virtual const Address::InstanceConstSharedPtr& directRemoteAddress() const PURE;
 
+  /**
```

Comment:
Just to leave an intermediate comment trail, can you add a TODO above here on what you plan to rename it to? Otherwise this will get confusing if someone looks at it now.

Reply:
yea, good point. let me add a TODO